mlrun/serving/routers.py

```diff
@@ -601,6 +601,24 @@ def do_event(self, event, *args, **kwargs):
                 if self.version:
                     response_body["model_version"] = self.version
                 response.body = response_body
+            elif name == self.name and event.method == "GET":
+                response = copy.copy(event)
+                response_body = {
+                    "name": self.name,
+                    "version": self.version or "",
+                    "inputs": [],
+                    "outputs": [],
+                }
+                for _, route in self.routes.items():
+                    response_random_route = route.run(copy.copy(event))
+                    if (
+                        response_random_route.body["inputs"] != []
+                        and response_random_route.body["outputs"] != []
+                    ):
+                        response_body["inputs"] = response_random_route.body["inputs"]
+                        response_body["outputs"] = response_random_route.body["outputs"]
+                        break
```

Comment:
Make it more flexible to take inputs from one model and outputs from another model to combine the full package: 

```suggestion
                    response_body["inputs"] = response_body["inputs"] or response_random_route.body["inputs"]
                    response_body["outputs"] = response_body["outputs"] or response_random_route.body["outputs"]
                    if response_body["inputs"] and response_body["outputs"]:
                        break
```